site/app/libraries/FileUtils.php

```diff
@@ -501,13 +538,25 @@ public static function validateUploadedFiles($files) {
         for ($i = 0; $i < $num_files; $i++) {
             //extract the values from each file
             $name = $files['name'][$i];
-            $type = mime_content_type($files['tmp_name'][$i]);
-            $size = $files['size'][$i];
-            $err_msg = "";
-
-            //did anything go wrong?
+            $tmp_name = $files['tmp_name'][$i];
+            $type = mime_content_type($tmp_name);
+            
+            $zip_status = FileUtils::checkZipFileStatus($tmp_name);
             $err_msg = ErrorMessages::uploadErrors($files['error'][$i]);
```

Comment:
If `$files['error'][$i]  !== UPLOAD_ERR_OK`, there's no point in bothering with trying to check zip file status.